devel/libjwt/Portfile

```diff
@@ -1,13 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-# Please do not update this Port to version 1.9 of libjwt.  1.8 is presently a dependency for the 'jwt' variant of the nginx port.
-# This Port will be updated as soon as the 'ngx-http-auth-jwt-module' project (https://github.com/TeslaGov/ngx-http-auth-jwt-module)
-# officially supports 1.9.
-
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        benmcollins libjwt 1.8.0 v
+revision            1
```

Comment:
I think `revision 1` must be a mistake, since you're bumping the version number of the package.

Reply:
I guess I didn't really understand the purpose of 'revision'.  I'll remove it.

Reply:
Do I need to squash my commits or just push the change?

Reply:
You need to squash your commits and push.

The purpose of `revision` is to distinguish two versions of the port that are based on the same upstream version. If a newer `revision` appears, it tells the system to replace the existing version.

Reply:
Okay - I'm utterly flummoxed (sorry - not a git master).  I've squashed my commits but pushing seems to be impossible as I'm changing a branch that's already been pushed.  Do I need to delete my upstream branch and re-push?  Been at this for over an hour so if you can offer any pointers, I'd be appreciative.

Reply:
You just use a `+` sign. As in

`git push origin +libjwt-1.9.0`

This tells git to force push the branch.